sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/ChatAsyncClient.java

```diff
@@ -291,7 +356,7 @@ public void addEventHandler(ChatEventType chatEventType, RealTimeNotificationCal
     }
 
     /**
-     * Remove handler from a chat event.
+     * Remove handler from a chat event for real-time notifications.
```

Comment:
nit; Let's agree to use one of `real time`, `real-time` or `realtime`. I think Firebase uses `realtime` from what I saw online.